cli/tools/test.rs

```diff
@@ -1054,7 +1054,7 @@ pub async fn run_tests_with_watch(
     let paths_to_watch_clone = paths_to_watch.clone();
 
     let maybe_resolver =
-      ps.maybe_import_map.as_ref().map(ImportMapResolver::new);
+      ps.maybe_import_map.clone().map(ImportMapResolver::new);
```

Comment:
It seems we have many different ways of getting a resolver... why aren't they all the same?

Reply:
I think this is also something we should fully address in #12684.  I will fix this one, but the problem is that we don't have the compat resolver everywhere at the moment, and I shouldn't address that as part of this PR.  Which means there is an inconsistency across the code base.  I cleaned it up a bit in the LSP (but the LSP doesn't support the compat resolver yet)